app/controllers/v0/sessions_controller.rb

```diff
@@ -82,7 +82,7 @@ def saml_user
 
     def create_saml_user
       user = User.new(user_attributes)
-      user = Decorators::MviUserDecorator.new(user).create unless user.loa1? || user.gender.nil?
+      user.mvi # caches mvi data
```

Comment:
If it always needs to be called, should we just do it in the User#initializer? Is there a reason to proactively cache it here as opposed to waiting for something to call it?